### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Lionel Pollien and Arthur Menétrey

### DISCUSSION THREAD

**Discussion by Xenogix** [heig-vd-dai-course/heig-vd-dai-course#64] [Show and tell]
Our group for the practical work 1 is composed of Pollien Lionel and Menétrey Arthur

**Comment by hadrylouis** [comment]
Could you please tell us a bit about what you plan to do for this practical work? :)

**Comment by poLionel** [comment]
We are doing a JSON-to-XML (and XML-to-JSON) converter !
